rhel/Dockerfile

```diff
@@ -1,9 +1,7 @@
-ARG RHEL_VERSION=8
+ARG RHEL_VERSION=7
```

Comment:
is the downgrade intended?

Reply:
yeah I upgraded as part of the PR and that broke tests. There were no security issues in 7 so going to keep us pinned there for now